MA-FE-NC-GAMES/components/VoteModifier.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+import { patchReviewVotes } from "./api";
+
+const VoteModifier = ({ votes, review_id }) => {
+	const [err, setErr] = useState(null);
+	const [currentVotes, setCurrentVotes] = useState(votes);
+
+	const handleUpvoteClick = () => {
+		setCurrentVotes((currentVotes) => +currentVotes + 1);
+		setErr(null);
+		patchReviewVotes(review_id, 1).catch((err) => {
+			setCurrentVotes((currentVotes) => currentVotes - 1);
+			setErr("Sorry, that didn't work, please try again");
+		});
+	};
+
+	const handleDownvoteClick = () => {
+		setCurrentVotes((currentVotes) => +currentVotes - 1);
+		setErr(null);
+		patchReviewVotes(review_id, -1).catch((err) => {
```

Comment:
Rather than a catch we want to check the message given back and see whether the message is the data (if successful) or an error message.  A catch block won't work here as we have sent a message on the back end if there is an error